daphne/src/vdaf/mod.rs

```diff
@@ -175,10 +180,17 @@ impl VdafConfig {
             return Err(DapError::Fatal("unexpected number of HPKE configs".into()));
         }
 
-        const N: usize = CTX_INPUT_SHARE.len();
-        let mut info = [0; N + 2];
-        info[..N].copy_from_slice(CTX_INPUT_SHARE);
-        info[N] = CTX_ROLE_CLIENT; // Sender role (receiver role set below)
+        let input_share_text = match version {
+            DapVersion::Draft02 => CTX_INPUT_SHARE_DRAFT02,
+            DapVersion::Draft03 => CTX_INPUT_SHARE_DRAFT03,
+            _ => panic!("unimplemented DAP version"),
```

Comment:
I wouldn't panic here, since the condition is triggered by user error (`version` is unhandled variant).